<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logs</class>
 <widget class="QDialog" name="Logs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>421</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReadLog">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>ReadLog</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QComboBox" name="LogSelection">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>40</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="LogSelectionlbl">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Log Name</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>511</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Exit</sender>
   <signal>clicked()</signal>
   <receiver>Logs</receiver>
   <slot>Exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ReadLog</sender>
   <signal>clicked()</signal>
   <receiver>Logs</receiver>
   <slot>Name()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Exit()</slot>
  <slot>Name()</slot>
 </slots>
</ui>
